SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -136,3 +204,22 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 
 	return PendingRequests;
 }
+
+void SpatialLoadBalanceEnforcer::QueueAclAssignmentRequest(const Worker_EntityId EntityId)
+{
+	UE_LOG(LogSpatialLoadBalanceEnforcer, Verbose, TEXT("Queueing ACL assignment request for entity %lld on worker %s."), EntityId, *WorkerId);
+	AclWriteAuthAssignmentRequests.Add(WriteAuthAssignmentRequest(EntityId));
+}
```

Comment:
I would probs just put these two lines at the bottom of the Maybe* method,

Reply:
I would prefer to have this as its own function as it feels more explicit. Not particularly strongly though, so if you do, will move